samples/bluetooth/mesh/sensor_client/prj.conf

```diff
@@ -47,6 +47,7 @@ CONFIG_BT_MESH_RELAY=y
 CONFIG_BT_MESH_FRIEND=y
 CONFIG_BT_MESH_TX_SEG_MAX=24
 CONFIG_BT_MESH_RX_SEG_MAX=16
+CONFIG_BT_MESH_RX_SEG_MSG_COUNT=2
```

Comment:
Why is it required?

Reply:
With the new sensor types, two segmented messages are being sent at the same time when client requests update or `get`. I kept getting the _No multi-segment message contexts available_ transport error.

Reply:
Or do one or both transactions happen over multicast address?

Reply:
I agree, this sounds strange and might be a bug. This started happening once you added the new sensors?

Reply:
Ok. I am not sure if this issue is related to the changes in this PR then. Could this be something to investigate in a separate task? The sample works like expected despite this error. To answer your question, it happens frequently and is easy to reproduce but it does not happen every time no.

Reply:
But does it happen without your addition of extra sensors? If yes, I'd say this is an unrelated bug and should be reported and fixed separately. If it only started happening after you changes, it should probably be fixed here

Reply:
I doubt misbehavior was added here, since @akredalen didn't change anything in transport.

Reply:
I tested the sample without any of my changes now, and I do see the occasional errors/warnings:
```
Time since presence detected(0): <value is not known>                                      
Chip temp: 20.25, total samples: 40                                                        
Chip temp: 20.25, total samples: 41                                                        
Ambient light level: 0                                                                     
Presence detected: 0                                                                       
Time since presence detected(0): <value is not known>                                      
[00:02:54.257,843] <wrn> bt_mesh_delayable_msg: Unable allocate 12 chunks, allocated 4     
Chip temp: 20.25, total samples: 42                                                        
[00:02:54.258,544] <wrn> bt_mesh_delayable_msg: Unable allocate 12 chunks, allocated 4     
[00:02:54.449,096] <err> bt_mesh_transport: No multi-segment message contexts available    
[00:02:54.458,282] <err> bt_mesh_transport: No multi-segment message contexts available    
Ambient light level: 0                                                                     
Presence detected: 0  
```

Reply:
I see messages. This is not related to lack of sar context or parallel sending of 2 sar transaction. This is lack of buffers for delayable messages. It requires adding more chunks `BT_MESH_ACCESS_DELAYABLE_MSG_CHUNK_COUNT`. Nevertheless, we can do better investigation in separate task

Reply:
Hmm. Would it make sense to increase the chunk count based on what was added in this PR or is it still a more general issue? I mean, it was already an issue before but it has certainly gotten worse with my latest changes. I would not mind adding the fix here if it makes more sense. If not I will create an issue on Jira. Let me know.